src/components/app/content/card/index.js

```diff
@@ -0,0 +1,56 @@
+import React, {Fragment, useState} from 'react';
+import './index.css'
+
+const classNames = require('classnames');
+
+function Card() {
+    const initHeader = 'Caption';
+    const initBody = 'Text...';
+
+    const [cardOptions, setCardOptions] = useState({checked: false, editable: false})
+    const [cardData, setCardData] = useState({header: initHeader, body: initBody})
+    const [backupState, setBackupState] = useState({header: initBody, body: initBody})
+
+    let cancelEditing = () => setCardOptions({editable: false, checked: cardOptions.checked})
+    let saveCardDataChanges = () => {
+        cancelEditing()
+        setBackupState({header: cardData.header, body: cardData.body})
+    }
+    let restoreCardData = () => {
+        cancelEditing()
+        setCardData({header: backupState.header, body: backupState.body})
+    }
+    let editCard = () => setCardOptions({editable: true, checked: false})
+    let selectCard = () => setCardOptions({checked: !cardOptions.checked, editable: cardOptions.editable})
+    let fillHeader = (event) => setCardData({header: event.target.value, body: cardData.body})
+    let fillBody = (event) => setCardData({header: cardData.header, body: event.target.value})
```

Comment:
All these functional expressions are immutable which requires using const for their declaration.